pkg/virt-handler/hotplug-disk/mount.go

```diff
@@ -42,12 +42,12 @@ var (
 	nodeIsolationResult = func() isolation.IsolationResult {
 		return isolation.NodeIsolationResult()
 	}
-	deviceBasePath = func(podUID types.UID) (*safepath.Path, error) {
-		return safepath.JoinAndResolveWithRelativeRoot("/proc/1/root", fmt.Sprintf("/var/lib/kubelet/pods/%s/volumes/kubernetes.io~empty-dir/hotplug-disks", string(podUID)))
+	deviceBasePath = func(podUID types.UID, kubeletPodsDir string) (*safepath.Path, error) {
+		return safepath.JoinAndResolveWithRelativeRoot("/proc/1/root", kubeletPodsDir, fmt.Sprintf("/%s/volumes/kubernetes.io~empty-dir/hotplug-disks", string(podUID)))
 	}
 
-	sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
-		return safepath.JoinAndResolveWithRelativeRoot("/proc/1/root", fmt.Sprintf("root/var/lib/kubelet/pods/%s/volumes", string(podUID)))
+	sourcePodBasePath = func(podUID types.UID, kubeletPodsDir string) (*safepath.Path, error) {
```

Comment:
Quick grep shows that `sourcePodBasePath` is not actually used in the code. It is defined here and then redefined in tests:
```
$ grep -RnI sourcePodBasePath
pkg/virt-handler/hotplug-disk/mount.go:49:	sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:72:	orgSourcePodBasePath   = sourcePodBasePath
pkg/virt-handler/hotplug-disk/mount_test.go:572:			sourcePodBasePath = orgSourcePodBasePath
pkg/virt-handler/hotplug-disk/mount_test.go:582:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:603:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:613:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:630:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:644:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:660:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:788:			sourcePodBasePath = orgSourcePodBasePath
pkg/virt-handler/hotplug-disk/mount_test.go:843:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
pkg/virt-handler/hotplug-disk/mount_test.go:973:			sourcePodBasePath = func(podUID types.UID) (*safepath.Path, error) {
```
I raised a PR to drop it https://github.com/kubevirt/kubevirt/pull/10216 (and also to clean up some other unused code). Probably, you do not need this change at all and can leave the function as it is. I think it will not affect your PR.